app/helpers/application_helper/toolbar/x_vm_center.rb

```diff
@@ -184,7 +184,8 @@ class ApplicationHelper::Toolbar::XVmCenter < ApplicationHelper::Toolbar::Basic
           :vm_chargeback,
           'fa fa-file-text-o fa-lg',
           N_('Show Chargeback preview'),
-          N_('Chargeback Preview')
+          N_('Chargeback Preview'),
+          :klass => ApplicationHelper::Button::VmPerf
```

Comment:
Is there an explanation of how this works?  If you don't provide a 5th argument to `button`, it's enabled by default?  Is it calling `disabled?` or `enabled?` on the klass specified here?  It seems like this is an easy thing to forget when adding a new button.

We don't have to fix this here but as someone unfamiliar with this, it's not obvious that's what this change is doing.

Reply:
Hi @jrafanie ,
if the 5th parameter is not provided, by default the button will be displayed.

As I understand, if a button (like these) needs to be displayed or not based on the record, we provide a `:klass` to validate the record. in this case, this helper has a `def disabled?` method which checks `if @record.has_perf_data?`